build.sbt

```diff
@@ -315,6 +316,10 @@ lazy val protobufV3 = akkaModule("akka-protobuf-v3")
     test in assembly := {}, // assembly runs tests for unknown reason which introduces another cyclic dependency to packageBin via exportedJars
     description := "Akka Protobuf V3 is a shaded version of the protobuf runtime. Original POM: https://github.com/protocolbuffers/protobuf/blob/v3.9.0/java/pom.xml")
 
+lazy val pemReader =
```

Comment:
I'm not too happy with this name and, instead, I think this could be `akka-pki`.

I'm avoiding anything with `TLS` or `SSL` in the name to keep this focused and avoid a name that's so generic that this ends up being used to store _anything_.

NOTE: the reason this package exists is to `akka-remote` doesn't have an extra dependency.

Reply:
My idea is to make this depend on `akka-remote` so it can provide an implementation of `SSLEngineProvider` that overrides the key and trust loading and uses the pem reader.

Reply:
Is this only intended to be used with akka-remote and not as a general utility from Akka HTTP or Akka gRPC also?

In that case I think we can just include it in akka-remote instead, with the external dependency marked as optional/provided. It can still be a SSLEngineProvider subclass there.

Reply:
The PEM reader was brought here for the cloudstate specific use case but being able to read PKCS#1 and PKCS#8 will be very valuable in any TLS use case and sample (e.g. https://github.com/akka/akka-grpc/blob/e460469331e7f2a8b8c58aa6a44cf3eac1cdb0eb/interop-tests/src/main/scala/akka/grpc/interop/AkkaGrpcServerScala.scala#L55-L84).

Reply:
ok, so keep the reader utility in new module, but then it shouldn't depend on akka-remote. The SSLEngineProvider can be in akka-remote with the provided dependencies.

Reply:
Regarding naming, I agree that pem-reader sounds very narrow. Can we broaden it somewhat so it could contain some eventual other utilities in the future? akka-pki might be good?